databuilder/databuilder/extractor/base_postgres_column_usage_extractor.py

```diff
@@ -0,0 +1,112 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+import abc
+import logging
+from collections import namedtuple
+from itertools import groupby
+from typing import (
+    Any, Dict, Iterator, Union,
+)
+
+from pyhocon import ConfigFactory, ConfigTree
+
+from databuilder import Scoped
+from databuilder.extractor.base_extractor import Extractor
+from databuilder.extractor.sql_alchemy_extractor import SQLAlchemyExtractor
+from databuilder.models.table_column_usage import ColumnReader
+
+TableKey = namedtuple('TableKey', ['schema', 'table_name'])
+
+LOGGER = logging.getLogger(__name__)
+
+
+class BasePostgresColUsageExtractor(Extractor):
```

Comment:
For several of the classes in this PR there is a `BasePostgres...Extractor` and a corresponding `Redshift...Extractor`. I see the potential reusability b/w pg and redshift but these base classes aren't actually unique to pg and redshift. In terms of semantics these appear to just be `BaseColUsageExtractor`, etc. that could be applied to any database. There is a [generic usage extractor](https://github.com/amundsen-io/amundsen/blob/main/databuilder/databuilder/extractor/generic_usage_extractor.py#L24), can the redshift extend that one?

Reply:
@sewardgw  , thanks for your review , I'll leverage https://github.com/amundsen-io/amundsen/blob/main/databuilder/databuilder/extractor/generic_usage_extractor.py and resubmit the PR